typescript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const width = 3;
+const height = 4;
+const area = width * height;
+console.log('value of area: ' + area);
+console.log('type of area: ' + typeof area);
+const bill = 20;
+const payment = 15.99;
+const change = bill - payment;
+console.log('value of change:' + change);
+console.log('type of change: ' + change);
+const quizzes = 90;
+const midterm = 100;
+const final = 100;
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade: ' + grade);
+console.log('type of grade' + typeof grade);
+const firstName = 'Abraham';
+const lastName = 'Razon';
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName: ' + fullName);
+console.log('type of fullName: ' + typeof fullName);
+const pH = 7;
+const isAcidic = pH <= 7;
+console.log('value of isAcidic: ' + isAcidic);
+console.log('type of isAcidic: ' + typeof isAcidic);
+const headcount = 300;
+const isSparta = headcount === 300;
+console.log('value of isSparta: ' + isSparta);
+console.log('type of isSparta' + typeof isSparta);
+const motto = fullName + ' is the GOAT';
```

Comment:
Close. This exercise is first asking to you assign `fullName` to `motto`. Then on another line using the addition assignment operator to concatenate the string ` is the Goat`.